src/modules/user/dtos/create-user.dto.ts

```diff
@@ -1,21 +1,15 @@
-import { IsArray, IsEmail, IsOptional, IsString } from 'class-validator';
-
-export class CreateUserDto {
-  @IsEmail()
-  email: string;
-
-  @IsString()
-  name: string;
-
-  @IsOptional()
-  @IsString()
-  phone?: string;
-
-  @IsOptional()
-  @IsString()
-  walletAddress?: string;
-
-  @IsArray()
-  @IsString({ each: true })
-  roleIds: string[];
-}
+import { z } from 'zod';
+
+export const CreateUserSchema = z.object({
+  email: z
+    .string({ message: 'wasteFootprint must be a string' })
```

Comment:
Here is email but the message is    " .string({ message: 'wasteFootprint must be a string' })"

Reply:
Correct -> email: z.string({
    message: 'email must be a string and should not be empty',
  }),